oe_theme.libraries.yml

```diff
@@ -1,23 +1,12 @@
 base:
   js:
     dist/scripts/ecl-ec-preset-website.js: {}
+    js/ecl_auto_init.js: {}
 style:
   css:
     theme:
       dist/styles/ecl-ec-preset-website.css: {}
       css/style.css: {}
-accordions:
```

Comment:
The library was removed but its still being loaded on the accordion.ui_patterns.yml file

Reply:
Very good catch.